### PR TITLE
Change the social icons border color

### LINKED ISSUE #5031: Change the social icons border color
Hi, this is a [first-timers-only issue](https://publiclab.github.io/community-toolbox/#r=all). This means we've worked to make it more legible to folks who either **haven't contributed to our codebase before, or even folks who haven't contributed to open source before**. 

If that's you, we're interested in helping you take the first step and can answer questions and help you out as you do. Note that we're especially interested in contributions from people from groups underrepresented in free and open source software!

We know that the process of creating a pull request is the biggest barrier for new contributors. This issue is for you 💝

If you have contributed before, **consider leaving this one for someone new**, and looking through our general [help wanted](https://github.com/publiclab/plots2/labels/help-wanted) issues. Thanks!

### 🤔 What you will need to know.

Nothing. This issue is meant to welcome you to Open Source :) We are happy to walk you through the process.

### 📋 Step by Step

- [ ] 🙋 **Claim this issue**: Comment below. If someone else has claimed it, ask if they've opened a pull request already and if they're stuck -- maybe you can help them solve a problem or move it along!

- [ ] 📝 **Update** . 
* The border and background of buttons in facebook and twitter buttons used is #4267b2 and #1b95e0 which is the main color of palette color followed by facebook and twitter. Let the background color remain same but Change the border hex to #29487d and ##00aced which is also a part of palette too but it shows a difference with background color.

Screenshot from 2019-03-12 22-55-25

*  update will be done in app/views/home/_social.html.erb

- [ ] 💾 **Commit** your changes

- [ ] 🔀 **Start a Pull Request**. There are two ways how you can start a pull request:

1. If you are familiar with the terminal or would like to learn it, [here is a great tutorial](https://egghead.io/series/how-to-contribute-to-an-open-source-project-on-github) on how to send a pull request using the terminal.

2. You can also [edit files directly in your browser](https://help.github.com/articles/editing-files-in-your-repository/) and open a pull request from there. 

- [ ] 🏁 **Done** Ask in comments for a review :)

### 🤔❓ Questions?

Leave a comment below!

### Is someone else already working on this?

We encourage you to link to this issue by mentioning the issue # in your pull request, so we can see if someone's already started on it. **If someone seem stuck, offer them some help!** Otherwise, [take a look at some other issues you can help with](https://publiclab.github.io/community-toolbox/#r=all). Thanks!

(This issue was created by [First-Timers-Bot](https://github.com/hoodiehq/first-timers-bot).)

### PR DESCRIPTION
**Pull request by daddyjab**
Changed `app/assets/stylesheets/dashboard.css` to change color of the borders on the Facebook and Twitter buttons:
1. Facebook button: Border changed from #4267b2 to #29487d
2. Twitter button: Border changed from #1b95e0 to #00aced

NOTE: The issue created indicated that the changes should be made to `views/home/_social.html.erb`, but I instead made the change to the CSS file indicated above because it seemed more appropriate to control the button border color via the CSS.  Also, I did not see a way to implement this border color change in the `_social.html.erb` file.

Fixes #5031 (<=== Add issue number here) 

Make sure these boxes are checked before your pull request (PR) is ready to be reviewed and merged. Thanks!

* [x] PR is descriptively titled 📑 and links the original issue above 🔗
* [x] tests pass -- look for a green checkbox ✔️ a few minutes after opening your PR -- or run tests locally with `rake test`
* [ ] code is in uniquely-named feature branch and has no merge conflicts 📁
* [x] screenshots/GIFs are attached 📎 in case of UI updation
Fixes #
* [x] ask `@publiclab/reviewers` for help, in a comment below

> We're happy to help you get this ready -- don't be afraid to ask for help, and **don't be discouraged** if your tests fail at first!

If tests do fail, click on the red `X` to learn why by reading the logs.

Please be sure you've reviewed our contribution guidelines at https://publiclab.org/contributing-to-public-lab-software 

Thanks!

2019-04-07 16_06_43-dashboard css - GitHub - Visual Studio Code
2019-04-07 16_08_13-🎈 Public Lab_ Research
2019-04-07 16_07_25-🎈 Public Lab_ Research
2019-04-07 16_06_43-dashboard css - GitHub - Visual Studio Code

### CODE REVIEW

**Comment by daddyjab** [comment]
@publiclab/reviewers:
Need your advice.  I'm trying to get my Pull Request approved, but not sure who is authorized to do this or the right procedure to pursue next.
* I forked the plots2 repo, make the needed change, and submitted this pull request to merge the change back into the main repo
* All of the automated checks have passed

Any guidance you can provide will be much appreciated. Thanks! - daddyjab

**Comment by gauravano** [comment]
Hey @daddyjab, thanks for your work, just want to add a suggestion that, it's good to make a separate feature-branch for the PR as that'll help you in keeping the master branch clean.

Thanks!

**Comment by gauravano** [comment]
@CleverFool77 changes seem good to you?

**Comment by gauravano** [comment]
Also, can you add the screenshot of before and after view of the buttons? Thanks!

File: app/assets/stylesheets/dashboard.css [hunk 1]
```diff
@@ -216,6 +216,20 @@
...
+   Revised to add border of
+   offsetting color to Twitter buttons */
+.dashboard .social-media-btn #follow-button {
+  border: 1px solid #00aced;
+}
+
+/* Issue #5031 (daddyjab):
+  Revised to add border of
+  offsetting color to Facebook buttons */
+.dashboard .social-media-btn #facebook button {
```
Comment (gauravano) [inline app/assets/stylesheets/dashboard.css]: Hey @daddyjab, please remove the comments.

Comment (daddyjab) [reply app/assets/stylesheets/dashboard.css]: Hi, @gauravano
Yes, I will add a "before" screenshot, and I will remove the comments from the CSS file changes.

Regarding the feature branch, I was not able to push to a feature branch on the plots2 main repo due to permissions.  But maybe you're saying I should create a feature branch on my _forked_ repo, which I'll do for these next changes- thanks!

**Comment by daddyjab** [comment]
Closing this pull request - will open a new one based off a feature branch associated with my forked repo. Also, found a defect in my original commit that I will fix with my new commit/pull request.

**Comment by gauravano** [comment]
Hey @daddyjab, can you drop a link to your new PR here, just for reference. Thanks!

**Pull request closed without merging**